concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+ID	NUMBER	ENGLISH	DATABASE_ID	SEMANTIC_CATEGORY	LIST	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Bowern-2021-203c-1	1	acorn bread	644	food	Culture Vocabulary		
+Bowern-2021-203c-2	2	acorn mush 	601	food	Culture Vocabulary		
+Bowern-2021-203c-3	3	afterworld, land of dead, Heaven 	603	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-4	4	alcohol 	652	food	Culture Vocabulary	404	ALCOHOL (FERMENTED DRINK)
+Bowern-2021-203c-5	5	armband 	764	dress	Culture Vocabulary		
+Bowern-2021-203c-6	6	arrow 	605	subsistence tool	Culture Vocabulary	977	ARROW
+Bowern-2021-203c-7	7	arrow straightener 	606	other	Culture Vocabulary		
+Bowern-2021-203c-8	8	awl 	765	subsistence tool	Culture Vocabulary	278	AWL
+Bowern-2021-203c-9	9	axe/stone axe 	707	subsistence tool	Culture Vocabulary	677	AXE
+Bowern-2021-203c-10	10	bag (string) 	766	culture-material	Culture Vocabulary		
+Bowern-2021-203c-11	11	bag, sack 	684	culture-material	Culture Vocabulary	2468	BAG
+Bowern-2021-203c-12	12	bait for fishing 	607	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-13	13	ball game 	602	food	Culture Vocabulary		
+Bowern-2021-203c-14	14	basin/bowl 	708	culture-material	Culture Vocabulary	915	SINK (BASIN)
```

Comment:
I think our SINK is more aimed towards a thing attached to a wall for washing your hands, while the basin/bowl here is more a PLATE: https://huntergatherer.la.utexas.edu/lexical/feature/708

Reply:
I will unmap this, then, as we have another (less ambiguous) PLATE in the same list.